front/tests/operational-studies.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { test, expect } from '@playwright/test';
+import { PlaywrightHomePage } from './home-page-model';
+
+// Describe the test suite for the home page of OSDR
+test.describe('Testing if all mandatory elements for configure a simulation are loaded in operationnal studies app', () => {
+  // Declare the necessary variable for the test
+  let playwrightHomePage: PlaywrightHomePage;
+
+  // This function will run before all the tests in this suite
+  test.beforeAll(async ({ browser }) => {
+    const page = await browser.newPage();
+    // Create an instance of the PlaywrightHomePage class
+    playwrightHomePage = new PlaywrightHomePage(page);
+    // Go to the home page of OSDR
+    await playwrightHomePage.goToHomePage();
+    await playwrightHomePage.goToStudiesPage();
+    await page.getByTestId('scenarios-filter-button').click();
+  });
+
+  // testing if all mandatory elements for configure a simulation are loaded
+  test('InfraSelector is displayed', async () => {
+    expect(playwrightHomePage.page.getByTestId('infraselector-button')).not.toEqual(null);
+  });
+
+  test('RollingStockSelector is displayed', async () => {
+    expect(playwrightHomePage.page.getByTestId('rollingdtock-selector')).not.toEqual(null);
+  });
+
+  test('TimetableSelector is displayed', async () => {
+    expect(playwrightHomePage.page.getByTestId('timetableSelector')).not.toEqual(null);
+  });
+
+  test('SpeedLimitSelector is displayed', async () => {
+    expect(playwrightHomePage.page.getByTestId('speed-limit-by-tag-selector')).not.toEqual(null);
+  });
```

Comment:
Use `test.describe` to group several test together (instead of using comments)

```suggestion
test.describe('testing if all mandatory elements for configure a simulation are loaded', () => {
  test('InfraSelector is displayed', async () => {
    expect(playwrightHomePage.page.getByTestId('infraselector-button')).not.toEqual(null);
  });

  test('RollingStockSelector is displayed', async () => {
    expect(playwrightHomePage.page.getByTestId('rollingdtock-selector')).not.toEqual(null);
  });

  test('TimetableSelector is displayed', async () => {
    expect(playwrightHomePage.page.getByTestId('timetableSelector')).not.toEqual(null);
  });

  test('SpeedLimitSelector is displayed', async () => {
    expect(playwrightHomePage.page.getByTestId('speed-limit-by-tag-selector')).not.toEqual(null);
  });
});
```